src/rmp/test/helpers.tcl

```diff
@@ -0,0 +1,89 @@
+# Helper functions common to multiple regressions.
```

Comment:
This should be a file link to the one in /test. maybe it is, but make sure it is not a copy